Overview/Overview of work done AutSPACEs Project.md

```diff
@@ -0,0 +1,125 @@
+
+### Work that I contributed during the internship period
+* I created a small MVP landing page to make myself understand the working of the Django. This landing consited of a picture and video explaining about the project. It also had an option for the user to record their email address for the community to get back to the users. The backend of the MVP consisted of all the Django modules and HTML components.
+
+**MVP Landing Page**
+![](https://i.imgur.com/AtMLzLp.png)
+* The next came splitting up the tasks into Static and dynamic components. I started creating the static pages according to the Figma wireframe. This page was designed with the help of HTML, Bootstrap and CSS. I was successfully able to understand and create a landing page, experiences page, Moderation page and also a sign up page (this was not part of the wireframe)
+
+**Landing Page**
+![](https://i.imgur.com/UIhIbJQ.png)
+
+**View Experiences Page**
+
+![](https://i.imgur.com/S1W18ti.png)
+
+** Moderation Page**
+![](https://i.imgur.com/WCXN5n6.png)
+
+**Sign Up Page**
+![](https://i.imgur.com/mkXWzf0.png)
+
+**User Sign Up Flow**
+
+![](https://i.imgur.com/PYr6pSY.jpg)
+
+* I also led the platform development team by presenting the work done during the community call meetups that occured every fortnight. A bief explaination of how the repository actually works was expained with the help of Miro Board. 
+![](https://i.imgur.com/SyXZ1y7.png)
+* I got help from various resources for Django, HTML, Bootstrap, CSS and also to understand the vast structure of Open Humans. 
+* For my Master's defense I had to submit a poster regading the work that I carried out during my internship in the form of the poster.
+![](https://i.imgur.com/7ytGxki.png)
+
+# Brief report
+### Introduction
+
+AutSPACEs is a citizen science platform co-created by autistic people and their supporters alongside researchers and the open source community. The aim is to create an online space where autistic people can share their experiences around sensory processing and autism. We know that the autistic people have challenges navigating the world that was not build by Autistic people and for the autistic people. We would like to use citizen science n a participatory framework to rise up the autistic voices and educate non-autistic people on how they can change the behavior and also change the world. We would like them to understand how to be more inclusive and enabling the autistic people. Understanding sensory processing differences, and learning what could make environments better for autistic people, are top community priorities. In order to really make a difference, we want to take the research out of the lab and learn from the lived experience of autistic people in their daily lives.
```

Comment:
We would like to use citizen science **_in_** a participatory framework to rise up....